js/src/forum/addSettingsItems.js

```diff
@@ -0,0 +1,110 @@
+import { extend } from 'flarum/extend';
+
+import SettingsPage from 'flarum/components/SettingsPage';
+
+import LoadingIndicator from 'flarum/components/LoadingIndicator';
+import Select from 'flarum/components/Select';
+import FieldSet from 'flarum/components/FieldSet';
+import Switch from 'flarum/components/Switch';
+
+import { SetTheme } from './setSelectedTheme';
+import fixInvalidThemeSetting from './fixInvalidThemeSetting';
+import GetTheme from './getTheme';
+import { Themes, Constants } from '../common/config';
+
+const LocalStorageKey = Constants.localStorageKey;
+
+export default function () {
+    extend(SettingsPage.prototype, 'settingsItems', function (items) {
+        const { user } = app.session;
+
+        const PerDevice = user.preferences().fofNightMode_perDevice ? user.preferences().fofNightMode_perDevice : false;
```

Comment:
I assume this is a boolean, so this can be simplified to just `user.preferences().fofNightMode_perDevice`, and if you truly just want a bool you can add a `!!` before it.